pylua/network.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import socket
+import tornado.httpserver
+import tornado.websocket
+import tornado.ioloop
+import tornado.web
+
+#websocket-client
+from websocket import create_connection
+
+class LuaWebSocketClient(object):
+    def __init__(self):
+        print '\nLuaWebSocketClient ctor\n'
```

Comment:
@OHerasym, use python logger instead of print. See https://github.com/malirod/pylua/blob/develop/pylua/test/test_json.py for the sample